services/wallet/transfer/commands.go

```diff
@@ -582,7 +582,10 @@ func (c *transfersCommand) processUnknownErc20CommunityTransactions(ctx context.
 	for _, tx := range allTransfers {
 		if tx.Type == w_common.Erc20Transfer {
 			// Find token in db or if this is a community token, find its metadata
-			_ = c.tokenManager.FindOrCreateTokenByAddress(ctx, tx.NetworkID, *tx.Transaction.To())
+			token := c.tokenManager.FindOrCreateTokenByAddress(ctx, tx.NetworkID, *tx.Transaction.To())
```

Comment:
This should be also run when seeing community on the network, see https://github.com/status-im/status-go/pull/4013/files